.travis.yml

```diff
@@ -4,11 +4,10 @@ before_install:
 rvm:
   - 2.4.2
```

Comment:
Can you add ruby 2.5 to the test matrix and update the version of 2.4?

See https://github.com/ManageIQ/manageiq/blob/master/.travis.yml#L5